pkgs/development/libraries/at-spi2-atk/default.nix

```diff
@@ -1,24 +1,47 @@
-{ stdenv, fetchurl, python, pkgconfig, popt, atk, libX11, libICE, xorg, libXi
-, intltool, dbus-glib, at-spi2-core, libSM }:
+{ stdenv
+, fetchurl
+
+, removeReferencesTo
+, meson
+, ninja
+, pkgconfig
+
+, at-spi2-core
+, atk
+, dbus
+, glib
+, libxml2
+
+, gnome3 # Only to pass update script
+}:
+
+with stdenv.lib;
 
 stdenv.mkDerivation rec {
-  versionMajor = "2.26";
-  versionMinor = "1";
-  moduleName   = "at-spi2-atk";
-  name = "${moduleName}-${versionMajor}.${versionMinor}";
+  name = "${pName}-${version}";
+  pName = "at-spi2-atk";
+  version = "2.26.2";
 
   src = fetchurl {
-    url = "mirror://gnome/sources/${moduleName}/${versionMajor}/${name}.tar.xz";
-    sha256 = "0x9vc99ni46fg5dzlx67vbw0zqffr24gz8jvbdxbmzyvc5xw5w5l";
+    url = "mirror://gnome/sources/${pName}/${versions.majorMinor version}/${name}.tar.xz";
+    sha256 = "0vkan52ab9vrkknnv8y4f1cspk8x7xd10qx92xk9ys71p851z2b1";
   };
 
-  nativeBuildInputs = [ pkgconfig intltool ];
-  buildInputs = [ python popt atk libX11 libICE xorg.libXtst libXi
-                  dbus-glib at-spi2-core libSM ];
+  nativeBuildInputs = [ removeReferencesTo meson ninja pkgconfig ];
+  buildInputs = [ at-spi2-core atk dbus glib libxml2 ];
 
-  doCheck = false; # needs dbus daemon
+  passthru = {
+    updateScript = gnome3.updateScript {
+      packageName = pName;
+      versionPolicy = "none";
+    };
+  };
 
-  meta = with stdenv.lib; {
+  meta = {
+    description = "D-Bus bridge for Assistive Technology Service Provider Interface (AT-SPI) and Accessibility Toolkit (ATK)";
+    homepage = https://gitlab.gnome.org/GNOME/at-spi2-atk;
+    license = licenses.lgpl2;
```

Comment:
Apparently, they use `lgpl21Plus` https://gitlab.gnome.org/GNOME/at-spi2-atk/blob/master/meson.build#L3

Reply:
But they do have: https://gitlab.gnome.org/GNOME/at-spi2-atk/blob/master/COPYING

Reply:
I would write BR to them.

Reply:
Right, it is `lgpl2Plus` according to the source code: https://gitlab.gnome.org/GNOME/at-spi2-atk/blob/0415628d74cffbde2c6f5a795f163e2126af4c79/atk-adaptor/accessible-stateset.c#L11

The meson file should be fixed.

Reply:
I wrote BR to them https://gitlab.gnome.org/GNOME/at-spi2-atk/issues/2